clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -115,6 +115,8 @@ Improvements to clang-tidy
 - Improved :program:`run-clang-tidy.py` script. Fixed minor shutdown noise
   happening on certain platforms when interrupting the script.
 
+- Improved :program:`clang-tidy` by accepting parameters file in command line.
```

Comment:
I'm missing the documentation of the actual feature in the main clang-tidy documentation.